src/window/mod.rs

```diff
@@ -0,0 +1,14 @@
+mod imp;
+
+use gtk::{gio, glib};
+
+glib::wrapper! {
+    pub struct Window(ObjectSubclass<imp::Window>)
+        @extends gtk::ApplicationWindow, gtk::Window, gtk::Widget, @implements gio::ActionMap, gio::ActionGroup;
+}
```

Comment:
```suggestion
        @extends gtk::ApplicationWindow, gtk::Window, gtk::Widget;
}
```

Not really useful here